### PR TITLE
tsukae: init at unstable-2021-04-19

### PR DESCRIPTION
**Pull request by L3afMe**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by legendofmiracles** [comment]
Result of `nixpkgs-review pr 122020` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tsukae</li>
  </ul>
</details>

**Review by legendofmiracles — APPROVED** [review]

**Review by moni-dz — APPROVED** [review]

File: pkgs/applications/misc/tsukae/default.nix [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+  runVend = true;
+
+  meta = with lib; {
+    description = "Show off your most used shell commands.";
+    homepage = "https://github.com/irevenko/tsukae";
+    license = licenses.mit;
+    mainProgram = pname;
+    maintainers = with maintainers; [ l3af ];
+    platforms = platforms.linux ++ platforms.darwin;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/tsukae/default.nix:24]:
```suggestion
```

buildGoModule defaults to this.

File: pkgs/applications/misc/tsukae/default.nix [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+  vendorSha256 = "sha256-mVhc9roT7rm9WMwck7jobjn9ykKQRgWJAqM75AmhMN0=";
+
+  runVend = true;
+
+  meta = with lib; {
+    description = "Show off your most used shell commands.";
+    homepage = "https://github.com/irevenko/tsukae";
+    license = licenses.mit;
+    mainProgram = pname;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/tsukae/default.nix:22]:
```suggestion
```

also the default.

Comment (L3afMe) [reply pkgs/applications/misc/tsukae/default.nix:22]: I was informed that there is currently a bug with packages versioned `unstable-YYYY-MM-DD`, which requires mainProgram to be specified. If that is false then I can remove it.

**Comment by legendofmiracles** [comment]
Squash all package related commits

**Pull request merged**